themes/default/Packages.template.php

```diff
@@ -500,11 +499,11 @@ function template_examine()
 
 	echo '
 	<div id="admincenter">
-		<h2 class="category_header">', $txt['package_examine_file'], '</h2>
-		<h2 class="category_header">', $txt['package_file_contents'], ' ', $context['filename'], ':</h2>
-		<div class="content">
-			<pre class="file_content">', $context['filedata'], '</pre>
-			<a href="', $scripturl, '?action=admin;area=packages;sa=list;package=', $context['package'], '">[ ', $txt['list_files'], ' ]</a>
+		<h2 class="category_header">', $txt['package_examine_file'], ' : ' , $context['package'], '</h2>
+		<h3 class="category_header">', $txt['package_file_contents'], ' ', $context['filename'], ':</h3>
+		<div class="content" style="font-size: 1.1em">
```

Comment:
Is that inline intended?

Reply:
Of course ... I mean you don't think I would be lazy and just add stuff like that do you ??? 😄

Reply:
Absolutely not. :joy_cat: